dist/s_bazel.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+
+import pprint
+
+platform2files = {}
+with open("filelist") as f:
+    for a in [l.split() for l in f if l != "\n" and not l.startswith("#")]:
```

Comment:
At least part of the functionality here (returning a list of source files) is available in `dist.py` via `source_files()`.